examples/prometheus/prometheus.yaml

```diff
@@ -619,6 +619,21 @@ objects:
           action: keep
           regex: openshift-template-service-broker;apiserver;https
 
+      # Scrape config for the service catalog
+      - job_name: 'openshift-service-catalog'
+        scheme: http
```

Comment:
we shouldn't be using insecure schemes for metrics endpoints

Reply:
I agree, and didn't even think of this before. Where are we collecting information about go-live requirements like this?